scripts/build.sh

```diff
@@ -0,0 +1,92 @@
+#!/usr/bin/env sh
+
+esbuild_cmd() {
+  format=$1
+
+  outdir="dist/esm"
+  platform="browser"
+
+  if [ "$format" = "cjs" ]; then
+    outdir="dist/cjs"
+    platform="node"
+  fi
+
+  ./node_modules/.bin/esbuild \
+    --outdir=$outdir \
+    --platform=$platform \
+    --target=es2019 \
+    --sourcemap \
+    --bundle \
+    --format=esm \
+    --splitting \
+    --log-level=error \
+    --jsx-factory='React.createElement' \
+    --jsx-fragment='React.Fragment' \
+    --loader:.json=json \
+    --loader:.js=jsx \
+    --external:"react" \
```

Comment:
Is there any guide on which packages / etc. we should place here?

Reply:
I could document these yes. Apart from `react` (which is a peerDependency), the others are Node.js modules imported by the dependencies. Same for `.json=json`, this is to support a connector importing a JSON file.

Reply:
Closing as the project is now using TSDX.